private-templates-service/client/src/store/templates/types.ts

```diff
@@ -0,0 +1,19 @@
+import { IncomingMessage } from "http";
+import { TemplateList } from "adaptive-templating-service-typescript-node";
+
+export interface TemplateState {
+  fetching: boolean;
+  templates: any;
+}
+
+//Action Types
+export const REQUEST_TEMPLATES = 'REQUEST_TEMPLATES';
+export const RECEIVE_TEMPLATES = 'RECEIVE_TEMPLATES';
+export const FAIL_GET = 'FAIL_GET';
```

Comment:
Can we change these to 
```
export const REQUEST_TEMPLATE_GET = 'REQUEST_TEMPLATE_GET';
export const REQUEST_TEMPLATE_GET_SUCCESS = 'REQUEST_TEMPLATE_GET_SUCCESS';
export const REQUEST_TEMPLATE_GET_FAIL = 'REQUEST_TEMPLATE_GET_FAIL';
```

You can change these values easily in VS code by right clicking on the variable name and selecting 'Rename Variable'

Reply:
As a part of this can you rename the action creator functions to 

```
export function requestTemplateGet(): TemplateAction {
export function requestTemplateGetSuccess(templates: TemplateList): TemplateAction {
export function requestTemplateGetFail(error: IncomingMessage): TemplateAction {
```

Should be able to rename these in similar fashion easily.

Reply:
I would go a step further and say `REQUEST_GET_TEMPLATES` so that we know this is for retrieving all the templates rather than just one.